src/CameraGroup.cpp

```diff
@@ -408,9 +404,7 @@ void CameraGroup::refineCameraGroupAndObjects(const int nb_iterations) {
                   if (this->id_ref_cam_ == cam_ptr->cam_idx_) {
                     refine_cam = false;
                   }
-                  for (int i = 0; i < obj_pts_idx.size(); i++) {
-                    cv::Point3f current_pts_3d =
```

Comment:
@rameau-fr `current_pts_3d` is never used. Is it correct behavior?

Reply:
same as before, can definitely remove if not used!